state/applier.go

```diff
@@ -9,9 +9,12 @@ import (
 	"github.com/filecoin-project/chain-validation/chain/types"
 )
 
+type MinerPenaltyFIL = abi.TokenAmount
+type GasRewardFIL = abi.TokenAmount
```

Comment:
I don't think these are necessary. As aliases, there's no actual safety, and they are the same logical type (tokens).

If you return a structure from ApplyMessage, I don't think any justification would remain.

Reply:
Agreed, removed.